### ISSUE TITLE
[Bug] An error occurred while adding router Asus RT-AX89X with error '10g' is not a valid PortType

### ISSUE THREAD

**Issue by Coolkids** [Vaskivskyi/ha-asusrouter#774]
### The problem

An error occurred while adding router Asus RT-AX89X with error '10g' is not a valid PortType

### Your device model

RT-AX89X

### Firmware type

Stock

### Firmware version

3.0.0.4.386_47468

### Integration version with the issue

0.28.0

### Method of the integration installation

HACS

### What version and type of Home Assistant installation do you use

Core 2024.1.2

### Is there anything useful in the logs?

```txt
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 406, in async_setup
    result = await component.async_setup_entry(hass, self)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/asusrouter/__init__.py", line 27, in async_setup_entry
    await router.setup()
  File "/config/custom_components/asusrouter/router.py", line 454, in setup
    await self._init_sensor_coordinators()
  File "/config/custom_components/asusrouter/router.py", line 827, in _init_sensor_coordinators
    available_sensors = await self.bridge.async_get_available_sensors()
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/asusrouter/bridge.py", line 226, in async_get_available_sensors
    SENSORS: await self._get_sensors_ports(),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/asusrouter/bridge.py", line 541, in _get_sensors_ports
    return await self._get_sensors(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/config/custom_components/asusrouter/bridge.py", line 488, in _get_sensors
    data = await self.api.async_get_data(datatype)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/asusrouter/asusrouter.py", line 644, in async_get_data
    processed = process(
                ^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/asusrouter/modules/endpoint/__init__.py", line 112, in process
    return submodule.process(data)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/asusrouter/modules/endpoint/ethernet_ports.py", line 41, in process
    port_type = PortType(port[0:3].lower())
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/enum.py", line 712, in __call__
    return cls.__new__(cls, value)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/enum.py", line 1135, in __new__
    raise ve_exc
ValueError: '10g' is not a valid PortType
```

**Comment by Vaskivskyi** [comment]
Hello. Sorry for the late reply. The issue will be fixed as soon as possible

**Comment by Vaskivskyi** [comment]
Hello. Sorry for the delay. The issue should be fixed in version `0.29.0` which is already available in HACS
